### PR TITLE
Feature/20220715/footer

### PR DESCRIPTION
**Pull request by NRaeewal**

### CODE REVIEW

**Comment by NRaeewal** [comment]
[footer](https://trello.com/c/aqjXqn7J/23-footer)

File: app/assets/stylesheets/common/layout.scss [hunk 1]
```diff
@@ -1,295 +1,352 @@
...
+
+        /* Primary/Darkest */
+
+        color: #004251;
+      }
+
+      .info {
+        /* Caption/ Light */
+
+        font-family: "Helvetica Neue";
```
Comment (AnnaBaranova) [inline app/assets/stylesheets/common/layout.scss:232]: @NRaeewal let's use typography base classes

File: app/assets/stylesheets/common/layout.scss [hunk 2]
```diff
@@ -1,295 +1,352 @@
...
+        font-weight: 700;
+        font-size: 12px;
+        line-height: 15px;
+        /* identical to box height */
+
+        letter-spacing: 0.003em;
+
+        /* Primary/Darkest */
+
+        color: #004251;
```
Comment (AnnaBaranova) [inline app/assets/stylesheets/common/layout.scss:226]: @NRaeewal use colorswatches

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -1,34 +1,55 @@
...
+   
+      <div class='links fgs-ali fgs-al fgs-al-h fgs-al-align-items-flex-start'>
+
+          <div class="company-links fgs-ali fgs-al fgs-al-v">
+          <span class="links-title fgs-ali">COMPANY</span>
+          <%= link_to "Services", services_path, :class => "fgs-ali" %>
+          <%= link_to "About", about_path, :class => "fgs-ali" %>
+          <%= link_to "Careers", careers_path, :class => "fgs-ali" %>
+          <%= link_to "Contact Us", contact_path, :class => "fgs-ali" %>
+          </div>
```
Comment (AnnaBaranova) [inline app/views/layouts/_footer.html.erb:30]: @NRaeewal I feel a lot of code in the erb file you transferred to footer.jsx. I don't think you need it here.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -6,6 +6,12 @@
 		<link rel="manifest" href="/manifest.json" />
 		<meta name="apple-mobile-web-app-capable" content="yes">
 		<%= include_gon %>
+		<%= javascript_pack_tag "footer" %>
```
Comment (AnnaBaranova) [inline app/views/layouts/application.html.erb:9]: @NRaeewal you mounted your component in footer.html.erb.

File: app/javascript/packs/footer.jsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+          </span>
+          <span className="info fgs-ali">info@bitesite.ca</span>
+        </div>
+      </div>
+
+      <div className="links fgs-al fgs-ali  fgs-al-h fgs-al-align-items-flex-start fgs-al-g-60 ">
+        <div className="company-links caption-medium fgs-ali fgs-al fgs-al-v fgs-al-g-6">
+          <span className="links-title fgs-ali">COMPANY</span>
+          <Link path="/services" className="fgs-ali">
+            Services
```
Comment (AnnaBaranova) [inline app/javascript/packs/footer.jsx:30]: @NRaeewal what do you think if all links would open a new tab? can we add this option?

**Review by AnnaBaranova — APPROVED** [review]

**Pull request merged**